app/components/footer/bootstrap.yaml

```
@@ -1,3 +1,5 @@
 _components:
   footer:
-    footerLinks: []
+    svgLogo: ''
+    logoUrl: ''
+    navegationLinks: []
```

Oops, I think there's a typo in here
```suggestion
    navigationLinks: []
```